hack/tools/logcheck/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains tool for checking use of unstructured logs in a package. It is created to prevent regression after packages have been migrated to use structured logs.
+
+**Installation:**`go install k8s.io/klog/hack/tools/logcheck`
+**Usage:** `$logcheck.go <package_name>`
+`e.g $logcheck $KUBE_ROOT/pkg/kubelet/lifecycle/`
```

Comment:
nit: $KUBE_ROOT no longer makes sense in the klog repo :smile:

Reply:
maybe just `./pkg/kubelet/lifecycle` ?

GOPATH is going away in go 1.17 anyhow, so users will have to start calling tools like this from within a module, where it will either be `k8s.io/kubernetes/pkg/kubelet/lifecycle` or `./pkg/kubelet/lifecycle`.